src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -88,6 +88,20 @@ const mockS3ReplyWithFile = async (
     filePath,
   );
 
+const mockS3ResponsesForBulkUploadProcessing = async (
```

Comment:
The refactor commit that introduces this method (341b6a0c3340653e233d19504ec9c3c511b83ddd) appears to be after a commit that references it (d532c2ab0cda35c085b3bd16d5ae54094ff76930).